examples/kitchen-sink/client/main.js

```diff
@@ -4,11 +4,15 @@ import * as layout from '@financial-times/dotcom-ui-layout'
 import * as appContext from '@financial-times/dotcom-ui-app-context'
 import * as tracking from '@financial-times/n-tracking'
 import * as ads from '@financial-times/n-ads'
+import * as clientEmbed from '@financial-times/dotcom-ui-client-embed'
 
 readyState.domready.then(() => {
   const flagsClient = flags.init()
   const appContextClient = appContext.init()
 
+  const clientEmbedClient = clientEmbed.init('custom-context')
```

Comment:
Probably should move this 'custom-context' constant into a 'constants.js' file as per convention in this repo...